### ISSUE TITLE
Newlines not correctly handled in product card titles

### ISSUE THREAD

**Issue by cathysarisky** [TryGhost/Ghost#20027]
### Issue Summary

If I press shift-enter while creating a title for a product card, then each line of the product title ends up in a separate span.  I can publish and it displays without issues.  (This is handy, because I might want to change how those spans are styled or laid out.)

Everything is OK until I open the post a second time in the editor.  At that point, the title is consolidated into a single span.

Maybe titles shouldn't be breakable into multiple spans, but it'd be nice if the behavior (whatever it is) were consistent, and opening a published post didn't reformat it.

### Steps to Reproduce

Create a product card.
Give it a title that spans multiple lines.
Publish. 
Observe the output of the post HTML, with the title in multiple spans.
Open the post in the editor again.  (Line breaks will be gone.)
Save, without making other changes.
Observe the output of the HTML, with the title now in a single span.

### Ghost Version

5.81.1

### Node.js Version

18

### How did you install Ghost?

WSL2/using Ghost-cli

### Database type

MySQL 8

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by nairvismaya** [comment]
can you assign me this

**Comment by gnimmelf** [comment]
Just a friendly reminder that this was explicitly stated as a feature.

https://forum.ghost.org/t/header-card-improvements-ghost-changelog/40545

https://forum.ghost.org/t/ghost-not-remembering-newlines-after-going-back-into-the-editor/48460/2

Hope it gets attention sonn.

**Comment by dhrcodeer** [comment]
Newlines are also not preserved in the headings and subheadings of a header card. It's the same behavior: it publishes and displays without issues, but on re-opening the post/page in the editor the newlines disappear.

Any updates on the issue?

**Comment by cathysarisky** [comment]
I have a PR open that fixes the behavior.  (It's in the Koenig repo, since this is actually a Koenig editor issue.)  Stay tuned! :)

**Comment by cathysarisky** [comment]
The PR to fix this behavior is over in the Koenig repo, and since it just got merged, I'm closing the issue as fixed! ref: https://github.com/TryGhost/Koenig/pull/1387

**Comment by muratcorlu** [comment]
@cathysarisky Is this fix rolled back? I see a revert commit in PR. With v5.103 I see this issue again.

@ronaldlangeveld I see that you reverted the changes. Can we learn the reason?

**Comment by cathysarisky** [comment]
Yeah, it got reverted because there was an error with rendering. I've got a new PR with a fix, but that hasn't yet been merged.
